custom/icds_reports/models/views.py

```diff
@@ -457,6 +457,8 @@ class AggAwcMonthly(models.Model):
         blank=True, null=True,
         help_text="Number of cases_person_all that are female between 15 and 18 years"
     )
+    cases_person_adolescent_girls_11_14_out_of_school = models.IntegerField(null=True)
+    cases_person_adolescent_girls_11_14_all_v2 = models.IntegerField(null=True)
```

Comment:
it would be nice to add help text for these fields, especially the v2, so that it is clear how it differs from v1